modules/rostests/apitests/shell32/closewnd.h

```diff
@@ -106,25 +83,9 @@ static inline VOID CloseNewWindows(PWINDOW_LIST pExisting, PWINDOW_LIST pNew)
         if (!IsWindowVisible(hWnd) || FindInWindowList(*pExisting, hWnd))
             continue;
 
-        SwitchToThisWindow(hWnd, TRUE);
-        WaitForForegroundWindow(hWnd); // SetForegroundWindow may take some time
-        DWORD_PTR result;
-        if (!SendMessageTimeoutW(hWnd, WM_SYSCOMMAND, SC_CLOSE, 0, SMTO_ABORTIFHUNG, 3000, &result) &&
-            !PostMessageW(hWnd, WM_SYSCOMMAND, SC_CLOSE, 0))
-        {
-            if (WaitForForegroundWindow(hWnd)) // We can't fake keyboard input if the target is not foreground
-            {
-                SendAltF4Input();
-                WaitForWindow(hWnd, IsWindowVisible, 1); // Closing a window may take some time
-            }
-
-            if (IsWindowVisible(hWnd))
```

Comment:
It might be of interest to keep this IsWindowVisible if-block, for diagnostics purposes.